src/api/app/views/webui/package/side_links/_assignments.html.haml

```diff
@@ -2,6 +2,10 @@
 - if assignment
   Assigned to:
   = link_to(assignment.assignee, user_path(assignment.assignee), class: 'mx-2')
+  = link_to(project_package_assignment_path(project, package, assignment), method: :delete,
```

Comment:
We shouldn't show this link to delete an assignment to users without permissions for deleting an assignment.

Reply:
Fixed